mappings/net/minecraft/enchantment/ImpalingEnchant.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_3588 net/minecraft/enchantment/ImpalingEnchant
```

Comment:
Should be renamed to `ImpalingEnchantment` for consistency with the other enchantments.

```suggestion
CLASS net/minecraft/class_3588 net/minecraft/enchantment/ImpalingEnchantment
```

Reply:
fixed in https://github.com/Legacy-Fabric/yarn/commit/0bb6d2624ea8ae13a35bc23e25852030671bdb39